Makefile.env.mk

```diff
@@ -51,10 +51,10 @@ IOT_HTTP_ADAPTER_IMAGE               ?= $(DOCKER_REGISTRY_PREFIX)$(DOCKER_ORG)/i
 IOT_MQTT_ADAPTER_IMAGE               ?= $(DOCKER_REGISTRY_PREFIX)$(DOCKER_ORG)/iot-mqtt-adapter:$(IMAGE_VERSION)
 IOT_LORAWAN_ADAPTER_IMAGE            ?= $(DOCKER_REGISTRY_PREFIX)$(DOCKER_ORG)/iot-lorawan-adapter:$(IMAGE_VERSION)
 IOT_SIGFOX_ADAPTER_IMAGE             ?= $(DOCKER_REGISTRY_PREFIX)$(DOCKER_ORG)/iot-sigfox-adapter:$(IMAGE_VERSION)
+IOT_TENANT_CLEANER_IMAGE             ?= $(DOCKER_REGISTRY_PREFIX)$(DOCKER_ORG)/iot-tenant-cleaner:$(IMAGE_VERSION)
```

Comment:
can you add this image to the lists below IMAGE_ENV and IMAGE_LIST thanks

Reply:
Thanks. I overlooked that. Fixed!